core/src/main/java/org/apache/carbondata/core/scan/result/BlockletScannedResult.java

```diff
@@ -145,6 +147,8 @@
 
   protected QueryStatisticsModel queryStatisticsModel;
 
+  protected LazyBlockletLoad lazyBlockletLoad;
```

Comment:
Actually I am confused with the name xxxLoad, why is it called load? I am wondering is there a common name for this technique used in presto?